### PR TITLE
25-Card1 and 16-Homepage

### PR DESCRIPTION
**Pull request by michael-greene-ey**
Worked on Card1 and Homepage in conjunction. Have some temp code included until I create the mock data in a different folder.

### CODE REVIEW

File: health-and-fitness-organizer/public/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```
Comment (ninohero) [inline health-and-fitness-organizer/public/index.html]: don't commit commented out code

File: health-and-fitness-organizer/src/components/Tagline.module.css [hunk 2]
```diff
@@ -2,5 +2,6 @@
   display: flex;
```
Comment (ninohero) [inline health-and-fitness-organizer/src/components/Tagline.module.css]: so Is the tagline component still used?  I saw above that it was commented out.  Is this file still necessary?

File: health-and-fitness-organizer/src/index.css [hunk 3]
```diff
@@ -1,13 +1,13 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
```
Comment (ninohero) [inline health-and-fitness-organizer/src/index.css]: so these fonts seem fairly random.  usually the max number of fonts you see in a font-family definition is 3.  it would go 

font-family: "specific font", moreGeneralFontName, generic-font-family. 

If you have to use that long a font definition, you could break the font definition into a variable just to clean up the code a little

`:root {
  --system-ui: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}

.element {
  font-family: var(--system-ui);
}

File: health-and-fitness-organizer/src/index.js [hunk 4]
```diff
@@ -3,6 +3,8 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
 
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (ninohero) [inline health-and-fitness-organizer/src/index.js:5]: this may seem silly but the line break on line 5 isn't necessary

File: health-and-fitness-organizer/src/pages/Homepage.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import Carousel from "react-bootstrap/Carousel";
+import CardHome from "../UI/CardHome";
+import workoutImg from "../Assests/images/workout.jpg";
+import healthContentImg from "../Assests/images/health-content.jpg";
+import styles from "./Homepage.module.css";
+import { bgcolor } from "@mui/system";
+
+const Homepage = () => {
+  const page = [
+    {
```
Comment (ninohero) [inline health-and-fitness-organizer/src/pages/Homepage.js:10]: you might want to consider keeping this data in a separate folder like I think you did in that other place

File: health-and-fitness-organizer/src/pages/Homepage.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      id: 2,
+      title: "Health Content",
+      description: "Managing and creating health content for you routine",
+      image: healthContentImg,
+    },
+  ];
+
+  return (
+    <div className={styles.homepage}>
+      <Carousel variant="dark" interval={null} indicators={null}>
```
Comment (ninohero) [inline health-and-fitness-organizer/src/pages/Homepage.js:27]: just wondering, are the null values necessary?  if you just didn't include the props would it do the same thing?

File: health-and-fitness-organizer/src/pages/Homepage.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        <Carousel.Item>
+          <CardHome
+            title={page[0].title}
+            description={page[0].description}
+            images={page[0].image}
+          />
+        </Carousel.Item>
+        <Carousel.Item>
+          <CardHome
```
Comment (ninohero) [inline health-and-fitness-organizer/src/pages/Homepage.js]: carousel slides is a good example of where you would probably want to have a loop to create the slide array rather than having them hard-coded.  it would pry be something like 

page.map((card, i) => {
   // render card
...

**Review by ninohero — CHANGES_REQUESTED** [review]
a few comments

also, I dont like the asset directory being capitalized ie: "Assets" should be "assets"

File: health-and-fitness-organizer/public/index.html (hunk 1 above)
Comment (michael-greene-ey) [reply health-and-fitness-organizer/public/index.html]: this is code that is included in create-react-app. I didn't want to delete it incase my code broke and I needed to debug the code. Should I delete it?

File: health-and-fitness-organizer/src/components/Tagline.module.css (hunk 2 above)
Comment (michael-greene-ey) [reply health-and-fitness-organizer/src/components/Tagline.module.css]: yes. At this point i don't think it's necessary. I will delete it.

File: health-and-fitness-organizer/src/index.css (hunk 3 above)
Comment (michael-greene-ey) [reply health-and-fitness-organizer/src/index.css]: this is some more code that is include in create-react-app. Since it's just fonts, I will delete it

File: health-and-fitness-organizer/src/pages/Homepage.js (hunk 5 above)
Comment (michael-greene-ey) [reply health-and-fitness-organizer/src/pages/Homepage.js:10]: this is code mock data code that will eventually go in a json folder. should I wait or move it into another folder for now?

File: health-and-fitness-organizer/src/pages/Homepage.js (hunk 6 above)
Comment (michael-greene-ey) [reply health-and-fitness-organizer/src/pages/Homepage.js:27]: yes, the default in Bootstrap is not null

File: health-and-fitness-organizer/public/index.html (hunk 1 above)
Comment (ninohero) [reply health-and-fitness-organizer/public/index.html]: yeah, delete it.  you can always go through the commit history if you needed it back

Comment (ninohero) [reply health-and-fitness-organizer/public/index.html]: manifest is It's a Web App Manifest that describes your application and it's used by e.g. mobile phones if a shortcut is added to the homescreen.

File: health-and-fitness-organizer/src/pages/Homepage.js (hunk 5 above)
Comment (ninohero) [reply health-and-fitness-organizer/src/pages/Homepage.js:10]: you can wait I guess.  the sooner you do it, though, the less tech debt you'd have to clean up.  and some tech leads don't like any tech debt to be committed

**Review by ninohero — APPROVED** [review]
